packages/core/test/hardhat/data-verification-mechanism/VotingV2.js

```diff
@@ -3988,6 +3988,81 @@ describe("VotingV2", function () {
     const finalContractBalance = await votingToken.methods.balanceOf(voting.options.address).call();
     assert.equal(finalContractBalance, "0");
   });
+  it("Staking during active reveal is dealt with correctly via pendingStakes", async function () {
+    // An issue was found in the contract wherein a staked voter stakes again during the active reveal round, thereby
+    // increasing their effective stake to a value higher than what they had at the variable freeze time (first commit).
```

Comment:
```suggestion
    // increasing their effective stake to a value higher than what they had at the variable freeze time (first commit or stake during active reveal phase).
```

In the test case the first variable freeze occurs with the stake